sw/device/tests/BUILD

```diff
@@ -36,6 +31,39 @@ opentitan_functest(
     ],
 )
 
+opentitan_functest(
```

Comment:
This is probably not the best approach to create these new binaries, should I create an issue for someone with more expertise in Bazel?